src/GameThumbnail.js

```diff
@@ -1,20 +1,28 @@
 import React from "react";
 import { Card, CardImg, CardBody, CardTitle } from "reactstrap";
 
-const changeCoverSide = (cover) => {
+const changeCoverSide = cover => {
```

Comment:
do you mean changeCoverSize ??

Reply:
did you mean "did" instead of "do" ?

by - Faculté d'anglais